### PR TITLE
Update SDB from git fixing several vulns

### PR DESCRIPTION
**Pull request by radare**

### CODE REVIEW

File: libr/include/r_types_base.h [hunk 1]
```diff
@@ -87,10 +87,13 @@ typedef struct _utX{
 #define UT32_HI(x) ((ut32)(((ut64)(x))>>32)&UT32_MAX)
 
 /* preventive math overflow checks */
-#define UT64_ADD_OVFCHK(x,y) ((UT64_MAX - x) <= y)
-#define UT32_ADD_OVFCHK(x,y) ((UT32_MAX - x) <= y)
-#define UT16_ADD_OVFCHK(x,y) ((UT16_MAX - x) <= y)
-#define UT8_ADD_OVFCHK(x,y) ((UT8_MAX - x) <= y)
+#if !defined(SZT_ADD_AOVFCHK)
```
Comment (alvarofe) [inline libr/include/r_types_base.h]: typo?

Comment (radare) [inline libr/include/r_types_base.h]: nope. on purpose. its also defined in sdb.h

Comment (radare) [inline libr/include/r_types_base.h]: omg (oh my glasses). yep :P

Comment (radare) [inline libr/include/r_types_base.h]: will fix in the next commit

**Pull request merged**